src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Abstractions/Mappers/MemberMapper.cs

```diff
@@ -13,7 +13,7 @@ public class MemberMapper : ElementMapper<ISymbol>
         /// <summary>
         /// The containg type of this member.
         /// </summary>
-        internal TypeMapper ContainingType { get; }
+        public TypeMapper ContainingType { get; }
```

Comment:
Making this public so that external consumers can also reason about the mapping inheritance structure. No need to only expose this internally given that everything else is public.